### ISSUE TITLE
Undefined method `allow_insecure_sign_in_after_accept=' for Devise:Module

### ISSUE THREAD

**Issue by boatrite** [scambra/devise_invitable#536]
This config option seems to be broken
```
$ cat Gemfile.lock | grep devise
    devise (3.4.1)
    devise_invitable (1.4.1)
      devise (>= 3.2.0)
  devise
  devise_invitable
```

`rails g devise_invitable:install` also failed to insert this config option into config/initializers/devise.rb. It only inserts up to the `invited_by_counter_cache` option

**Comment by SupremeA** [comment]
I agree cant find it and cant seem to pass it into the devise options within the model

**Comment by Pipinha** [comment]
Hi guys,

Use "Devise.allow_insecure_sign_in_after_confirmation" instead "Devise.allow_insecure_sign_in_after_accept"

**Comment by SupremeA** [comment]
If you are saying to pass that into the devise options in the model I just tried and it still doesnt work.

**Comment by SupremeA** [comment]
Yes, I see it...can I put it in the Devise initalizer like this?

  config.allow_insecure_sign_in_after_confirmation = true

**Comment by Pipinha** [comment]
if is a general rule it's ok ;)

**Comment by SupremeA** [comment]
ok thanks Im still not getting it but I think it may have something to do with my strong parameters because i keep getting this (below) so im going to try and figure that out

    Processing by Admins::SessionsController#create as HTML
    #it passes the parameters
    #then says "Completed 401 Unauthorized" 

Any ideas on that?

**Comment by Pipinha** [comment]
Do you have some method running before creating the user?

Check this answer...it could help you
http://stackoverflow.com/questions/15864554/check-if-user-is-active-before-allowing-user-to-sign-in-with-devise-rails

**Comment by SupremeA** [comment]
Thanks for your help. I think my issue is in the invitation accept. Its not passing anything other than the invitation_token, I have created a user but when I look at the console that user has no password so I cant log them in. That seems to be the issue.
